package.json

```diff
@@ -96,6 +100,7 @@
         "prettier": "2.1.1",
         "react-test-renderer": "16.13.1",
         "ts-jest": "26.3.0",
+        "ts-node": "^9.0.0",
```

Comment:
Pin dependency?

I believe sometime, somewhere in the future as if we were a big company, we might have a DevOps repo with a set of rules that execute as CI pipelines and ensure things like this (dependencies pinning, prettify, localize, domain/data mixed imports, etc...) and auto commits to the branch to fix them.